build.gradle.kts

```diff
@@ -310,20 +335,50 @@ fun Project.setupPublishing(isLibrary: Boolean) {
 
     tasks.withType<ArtifactoryTask> { publications(publicationName) }
 
+    apply(plugin = "com.jfrog.bintray")
```

Comment:
Did we find out if the order matters here or not? If so, we should add a comment.